php/Blocks/Loader.php

```diff
@@ -473,19 +486,25 @@ protected function block_template( $name, $type = 'block' ) {
 
 			// This is not a load once template, so require_once is false.
 			load_template( $theme_template, false );
-		} else {
-			if ( ! current_user_can( 'edit_posts' ) || ! isset( $templates[0] ) ) {
-				return;
-			}
-			// Hide the template not found notice on the frontend, unless WP_DEBUG is enabled.
-			if ( ! is_admin() && ! ( defined( 'WP_DEBUG' ) && WP_DEBUG ) ) {
-				return;
-			}
+			return;
+		}
+
+		if ( ! empty( $this->blocks[ "genesis-custom-blocks/{$name}" ]['templateMarkup'] ) ) {
+			$this->template_editor->render_markup( $this->blocks[ "genesis-custom-blocks/{$name}" ]['templateMarkup'] );
```

Comment:
This renders the markup:

![renders-markup](https://user-images.githubusercontent.com/4063887/117106816-22e9dd80-ad46-11eb-9007-72193785ef9d.png)